platform/linux-generic/odp_pool.c

```diff
@@ -348,7 +348,10 @@ static odp_pool_t pool_create(const char *name, odp_pool_param_t *params,
 		break;
 
 	case ODP_POOL_PACKET:
-		headroom    = CONFIG_PACKET_HEADROOM;
+		if (params->pkt.min_headroom < CONFIG_PACKET_MAX_HEADROOM)
+			headroom = params->pkt.min_headroom;
+		else
+			headroom    = CONFIG_PACKET_MAX_HEADROOM;
```

Comment:
Trying to specify a headroom > `CONFIG_PACKET_MAX_HEADROOM` should fail the call, as noted earlier.

Reply:
It's better not to change headroom implementation in this set but just implement the capability == return the current headroom as the max. Other parts of the code assume that init time headroom is constant:

return CONFIG_PACKET_HEADROOM + (head - base);

Reply:
@psavol I'm not sure what you're saying here. This PR has two commits: One for the API changes and one to change the odp-linux pool implementation to reflect these new API changes. Are you suggesting that we ignore the API changes?

Reply:
@psavol CONFIG_PACKET_HEADROOM is set as default in odp_pool_params_init() function. So it should not break any existing implementation.

Also do you mean to set CONFIG_PACKET_HEADROOM as the maximum headroom in capability.